companies/hm.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "hm",
```

Comment:
This is also a pretty big company seated in almost every country in the world with the HQ in Sweden. I am not sure if we have other HMs in the database already, but the slug seems a little general since this is apparently only relevant in gb from your request.

Reply:
We have one for Germany. Our schema (and potentially my approach to curating) is somewhat limited in that regard.

I think we talked about this a while ago but with these multi-national companies, I often have to decide between putting them into a single record or splitting them into one for each country (or even group only some countries together but not others…). The reason is that while they are the same company, they often have different attributes (address, phone number, email even) depending on the country.  
My decision is usually based on a) the convenience for the user (having a local point of contact is much better, of course) vs. the convenience for me (let's face it, creating a record each for n (with n fairly large) countries is quite annoying :D) and b) whether I can reasonably assume that there is a single country that is mostly responsible for handling privacy requests.

The 'ideal' solution here in my opinion would probably be to also allow a `country` => `value` dict for most attributes, i.e. we could either have `"email": "dataprotection@hm.com"` or `"email": { "gb": "dataprotection.gb@hm.com", "de": "dataprotection.de@hm.com", "fr": "dataprotection.fr@hm.com" }`, maybe even with an optional fallback (`"other": "dataprotection.gb@hm.com"`).

Reply:
The problem with that solution of course being that a) that makes both the schema and our code a lot more complex and b) we would have to go back and amend many of the old records (although that is not that bad as that schema would be entirely backwards-compatible, the older records just wouldn't benefit from the new options).